l1-contracts/contracts/bridge/ntv/L1NativeTokenVault.sol

```diff
@@ -224,14 +235,48 @@ contract L1NativeTokenVault is IL1NativeTokenVault, IL1AssetHandler, NativeToken
         }
     }
 
+    function _deployBridgedToken(
+        uint256 _originChainId,
+        address _originToken,
+        bytes memory _erc20Data
+    ) internal virtual override returns (address, uint256) {
+        originChainId[DataEncoding.encodeNTVAssetId(_originChainId, _originToken)] = _originChainId;
```

Comment:
is it the same as on L2 ntv?

Reply:
No, the _originChainId is the origin of the message, which might not be the origin of the token
On the l2 we get the tokenOriginChainId returned from the bridgeInit